src/applications/vaos/tests/containers/TypeOfCarePage.unit.spec.jsx

```diff
@@ -73,7 +69,6 @@ describe('VAOS <TypeOfCarePage>', () => {
 
     expect(updateFormData.firstCall.args[2].typeOfCareId).to.equal('323');
     form.unmount();
-    resetFetch();
```

Comment:
Instead of removing this, can you call `global.fetch.resetHistory()`? I think you're correct to want to remove `resetFetch`, it's not doing what we want here, but `resetHistory` will do some of the cleanup we want and might avoid some issues with other tests.

Reply:
Done